### DISCUSSION TITLE
Have "parent" recipe pull ingredient lists from included "recipes"

### DISCUSSION THREAD

**Discussion by MattSidney** [TandoorRecipes/recipes#2225] [Ideas]
I am creating a "Christmas Turkey" recipe at the moment. When doing this I am adding a number of "child" recipes; "herb may", "sage sausage stuffing", "Spatchcocked Turkey". 
I am adding these in as a Recipe under individual steps which is working great. I am also able to create a shopping list from the parent "Christmas Turkey" recipt it seems that pulls ingredients from each of these "child" recipes. 
However, within the "Christmas Turkey" recipe the "Ingredient Overview" does not pull in the ingredients from the "child" recipies. 

Is this intended? If so, would it be possible to request an option to do the opposite and actually display all ingredients from the "child" recipies?

**Comment by vabene1111** [comment]
#2212
